images/Dockerfile

```diff
@@ -184,7 +177,111 @@ COPY --from=xrootd-plugin-builder /usr/include/nlohmann /usr/include/nlohmann
 # Copy the JSON schema validator library
 COPY --from=xrootd-plugin-builder /usr/lib64/libnlohmann_json_schema_validator.a /usr/lib64
 
-RUN chmod +x /pelican/osdf-client \
+# Install tini
+RUN curl -o tini https://github.com/krallin/tini/releases/download/v0.19.0/tini-static-amd64 &&\
+    chmod +x tini &&\
+    mv tini /bin/tini
```

Comment:
tini is available in the EPEL repos; you should install it via yum (probably in the 'dependency-build' stage that this stage inherits from).